Jantimizer/ExperimentSuite/SuiteDatas/SuiteDataSets.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using DatabaseConnector.Connectors;
```

Comment:
Why havent you added a new one for this experiment with dynamic depth?
Should there not also be a `GetPostgresSD_DynDepth` and `GetMySQLSD_DynDepth`?